tests/util.go

```diff
@@ -39,6 +40,15 @@ const (
 	paramFlag            = "-p"
 )
 
+func CreateDirIfNotExist(dir string) {
+	if _, err := os.Stat(dir); os.IsNotExist(err) {
```

Comment:
I believe this call to Stat is not needed because `MkDirAll` does not return an error if the dir already exists.